index.html

```diff
@@ -57,6 +60,164 @@ <h1 id="primary-text">
+                            <li class="tech-card">
+                                <p>Ruby on Rails</p>
+                            </li>
+                            <li class="tech-card">
+                                <p>CSS</p>
+                            </li>
+                            <li class="tech-card">
+                                <p>JavaScript</p>
+                            <li class="tech-card">
+                                <p>HTML</p>
+                            </li>
+                        </ul>
+                        <div class="bottom">
+                            <button class="see-btn">
+                                <p>See Project</p>
+                            </button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <section id="about-me">
+            <div id="ab-me-img"></div>
+            <div id="ab-me-1-img"></div>
+            <div id="ab-me-desktop-img"></div>
+            <div id="ab-me-desktop-1-img"></div>
+
+            <h1 class="title">About me</h1>
+            <p class="text">
+                I can help you build a product , feature or website Look through some of my work and experience! If you
+                like
+                what you see and have a project you need coded,
+                don’t hesitate to contact me.
+            </p>
+            <button class="see-btn">
+                <p class=".text">
+                    Get my resume
+                </p>
+            </button>
+        </section>
+
+        <section id="skills">
+            <div class="skill-card">
+                <div class="skill-icon-ctn">
+                    <img class="skill-icon" src="icons/languages.svg" />
+                </div>
+                <p class="skill-text">Languages</p>
+                <ul class="skill-list">
+                    <li>JavaScript</li>
+                    <li>Ruby</li>
+                    <li>HTML</li>
+                    <li>CSS</li>
+                </ul>
+            </div>
+            <div class="skill-card">
+                <div class="skill-icon-ctn">
+                    <img class="skill-icon" src="icons/frameworks.svg" />
+                </div>
+                <p class="skill-text">Frameworks</p>
+                <ul class="skill-list">
+                    <li>JavaScript</li>
+                    <li>Ruby</li>
+                    <li>HTML</li>
+                    <li>CSS</li>
+                </ul>
+            </div>
+            <div class="skill-card">
+                <div class="skill-icon-ctn">
+                    <img class="skill-icon" src="icons/skills.svg" />
+                </div>
+                <p class="skill-text">Skills</p>
+                <ul class="skill-list">
+                    <li>JavaScript</li>
+                    <li>Ruby</li>
+                    <li>HTML</li>
+                    <li>CSS</li>
+                </ul>
```

Comment:
- [ ] Please make sure you are putting` skills` in the `Skills` category like API design, Version control etc,  as mentioned in the design

|Figma design|Your design|
|----|-----|
|<img width="536" alt="Screenshot 2023-06-05 at 11 09 57 PM" src="https://github.com/Movgto/my-portfolio/assets/44798044/b4d157e3-afbc-48ef-8232-1fee3f1ac1ec">|<img width="307" alt="Screenshot 2023-06-05 at 11 06 54 PM" src="https://github.com/Movgto/my-portfolio/assets/44798044/d2878fbe-3492-422b-a688-ae33876a9f9b">|